### PR TITLE
Hello world Microverse

### PR DESCRIPTION
**Pull request by arnold-O**
features added include:
.Linters
.index.HTML
.index.CSS
.readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
```
Comment (Shedrack-Sunday) [inline README.md:27]: Kindly take this comment section out to keep the code base clean 👍

File: README.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (Shedrack-Sunday) [inline README.md:31]: - [ ] If you notice, there's an instruction to replace the logo with yours or remove it entirely. This is to personalise the project to be your ow work and make it look professional. good job 👏

File: README.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul> -->
+</details>
+
+<details>
+<!-- <summary>Database</summary> -->
+  <!-- <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul> -->
+</details>
```
Comment (Shedrack-Sunday) [inline README.md:92]: - [ ] I Suggest you edit the technologies here t the ones you made use of. Such as `HTML, CSS `, etc

File: README.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/arnold-O)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/arnoldoghiator/)
```
Comment (Shedrack-Sunday) [inline README.md:169]: -  [ ] If you check the [README TEMPLATE](https://github.com/microverseinc/readme-template) Provided, you seem to be missing your twitter handle. Kindly to add it up 👍

File: README.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
```
Comment (Shedrack-Sunday) [inline README.md:179]: - [ ]  As a form of good practice, please personalize the description on line `178`, as it serves only as a guide for you.
- [ ] Include future features the project might have, this is useful in future portfolio projects to give your viewers a sense of what you are building

File: README.md [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (Shedrack-Sunday) [inline README.md:219]: - [ ] What you have at the root of your project is a `MIT.md` file. kindly fix it to make the license accessible 💯

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @arnold-O  , 

You did a good job so far! 👏 
However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Good commit messages
✅ Good PR summary



## Required Changes ♻️
- [x] Please make sure to fill up the readme template with all the features provided.  I've listed the items omitted. It will make your work look  professional and well  documented
- [ ] You linters are not passing. I notice this is as a result of a conflict you have with this pull request kindly fix it and make sure they checkout as ✔️ .

- [ ] All sections with the `>` sign are guidelines for you. you should please edit it appropriately 

- [x] Check below to see a more detailed review of what is needed. good job 👏 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Shedrack-Sunday** [comment]
> MIT LICENSE was added, please review it as well

Hello @arnold-O, please re-summit on your dashboard and it will be treated. 

PS: Fix your Linters. They have to be all green 😃

File: README.md (hunk 6 above)
Comment (Favourezeugwa) [reply README.md:177]: - I am not sure you are referencing your MIT.md License properly and this has caused the link to break. Just as the previous reviewer said, Kindly replace  `(./LICENSE)` to `(MIT.md)` as that's the name of the  License file in your project's file structure and not `./LICENSE`

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @arnold-O , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-brianna


## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is (MIT.md) licensed.
+
+
```
Comment (osamaashraf6) [inline README.md:178]: - [ ] You did well 😃at adding the MIT LICENCE in the folders of the project, but when I see your readme I found that you put the MIT link as a text, so please fix this as stated by the previous reviewer😊 by adding this line at the MIT LICENCE like that and when you try to make a link use like this ``` [name of link](the URL)``` like I did I put your link inside the round braces ,so became a link😊
 ``` This project is [MIT](https://github.com/arnold-O/Hello_world_microverse/blob/feature/MIT.md) licensed.``` to let the users when they click on the MIT link redirect them to the page of LICENCE💯

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi  @arnold-O ☺️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

SoCloseSoCloseSethMeyersGIF (2)


## Highlights ✔️

- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ]  Please consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, for example: "Fix Licence " instead of "licence fixed" or  ... etc 👍


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @arnold-O , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
nicee

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] The changes as request by previous reviewer were implemented :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**